packages/block-editor/src/components/unit-control/stories/index.story.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import UnitControl from '../';
+
+const meta = {
+	title: 'BlockEditor/UnitControl',
+	component: UnitControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'UnitControl allows the user to set a numeric quantity as well as a unit ',
+			},
+		},
+	},
+	argTypes: {
+		onChange: {
+			action: 'onChange',
+			description: 'Callback function when the value changes.',
+			table: {
+				type: { summary: 'function' },
+			},
+		},
+		onUnitChange: {
+			action: 'onUnitChange',
+			description: 'Callback function when the unit changes.',
+			table: {
+				type: { summary: 'function' },
+			},
+		},
+		labelPosition: {
+			control: 'radio',
+			options: [ 'top', 'side', 'bottom' ],
+			description: 'The position of the label.',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'top' },
+			},
+		},
+		label: {
+			control: 'text',
+			description: 'The label for the control.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		value: {
+			control: 'text',
```

Comment:
```suggestion
			control: { type: null },
```

I believe the control is not necessary since the value can be updated directly via the UI.